lnwallet/channel.go

```diff
@@ -3862,7 +3856,7 @@ func (lc *LightningChannel) RevokeCurrentCommitment() (*lnwire.RevokeAndAck, []c
 // commitment, and a log compaction is attempted. In addition, a slice of
 // HTLC's which can be forwarded upstream are returned.
 func (lc *LightningChannel) ReceiveRevocation(revMsg *lnwire.RevokeAndAck) (
-	[]*PaymentDescriptor, error) {
+	*channeldb.FwdPkg, []*PaymentDescriptor, []*PaymentDescriptor, error) {
```

Comment:
The `godoc` comment wasn't updated to describe the additional return parameters (now the atomic forwarding unit, and the the adds+settles).